src/fields.js

```diff
@@ -0,0 +1,18 @@
+// This file is to store all fields that we want to list
+// If we add hidden parameter to a field
+// the hidden fields would be only render in the field selection dropdown menu
+// :Param collapse: true -> close , false or default undefined -> open
+// :Param hidden: true -> hide the field, false or default  undefined-> show the field
+
+export const fields = [
+  { label: "All text fields", field: "searchText", type: "text" },
+  { label: "Identifier", field: "id", type: "text" },
+  { label: "Source", field: "source", type: "list-facet", facetSort: "index", collapse: true },
+  { label: "Context", field: "hasContextCategory", type: "list-facet", facetSort: "count", collapse: true },
+  { label: "Material", field: "hasMaterialCategory", type: "list-facet", facetSort: "count", collapse: true },
+  { label: "Specimen", field: "hasSpecimenCategory", type: "list-facet", facetSort: "count", collapse: true },
+  { label: "Registrant", field: "registrant", type: "list-facet", facetSort: "count", collapse: true },
+  { label: "Sampling Purpose", field: "samplingPurpose", type: "list-facet", facetSort: "index", hidden: true, collapse: true },
```

Comment:
Why is this one `hidden: true` *and* `collapse: true`?

Reply:
These are two control parameters. `hidden` is defined by us to select fields. `collapse` is defined by `solr-faceted-search-react` to control the facet toggle event. This morning, we mentioned closing the facet toggle for `list-facet` and `range-facet`. collapse = true to close the toggle.

Reply:
Right, but this field is different from the others (they don't have `hidden: true`).  Why is it different?

Reply:
I just set `samplingPurpose` and `keywords` as `hidden: true` for the test. We probably need to discuss which fields should be included and set as `hidden: true`.

Reply:
I see.  We should remove that before merging, right?

Reply:
Yeah